docs/source/general/predictions.rst

```diff
@@ -15,19 +15,20 @@ You can pass in a sample of data (image file path, a string of text, etc) to the
 	
 .. code-block:: python
 
-    from flash import Trainer
-    from flash import download_data
-    from flash.vision import ImageClassificationData, ImageClassifier
+	from flash import Trainer
```

Comment:
IDK if this matters but these code-blocks have more indentation than others

Reply:
yeah this was an issue I had to remove previously